src/sourceTracking.ts

```diff
@@ -5,11 +5,10 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import * as fs from 'fs';
-import { isAbsolute, relative, resolve, sep, normalize } from 'path';
-import { EOL } from 'os';
-import { NamedPackageDir, Logger, Org, SfProject } from '@salesforce/core';
+import { resolve, sep, normalize } from 'path';
+import { NamedPackageDir, Logger, Org, SfProject, Lifecycle } from '@salesforce/core';
 import { AsyncCreatable } from '@salesforce/kit';
-import { getString, isString } from '@salesforce/ts-types';
+import { getString, isString, getBoolean } from '@salesforce/ts-types';
```

Comment:
Are we able to get rid of `getString` and `getBoolean`?

Reply:
yes!